packages/commonwealth/server/sequelize.json

```diff
@@ -1,4 +1,11 @@
 {
+  "test": {
+    "username": "commonwealth",
+    "password": "edgeware",
+    "database": "common_test",
+    "host": "127.0.0.1",
+    "dialect": "postgres"
+  },
   "development": {
```

Comment:
I was wondering if this config json is a better replacement of all the initial configuration we are doing in database.ts

Reply:
It's not the same. This json file is used by `sequelize-cli` (i.e. migrations) for creating/drop/connecting databases. `database.ts` is what is actually used by the node application.